R/lassnags.r

```diff
@@ -218,28 +219,29 @@ lassnags_wing = function (las, neigh_radii = c(1.5,1,2), low_int_thrsh = 50, upp
 
   # ====== STEP 2 =======
   # Point classificaitons based on rough interpretation of Table 2 - pg. 172
-  # Vlaues supplied/specified by user in BranchBolePtRatio.mat
+  # values supplied/specified by user in bbpr_thresholds
 
   verbose("Classifiying points...")
+  las@data[, snagCls :=
   ifelse(sph_PtDen>=pt_den_req  & sph_BranchBolePtRatio_mean>=bbpr_thresholds[1,1] &
          sm_cyl_PtDen>=pt_den_req & sm_cyl_BranchBolePtRatio_mean>=bbpr_thresholds[2,1] &
          lg_cyl_PtDen>=pt_den_req & lg_cyl_BranchBolePtRatio_mean>=bbpr_thresholds[3,1],
-         las@data[, SnagCls := 1],        # General snag class
+         1,        # General snag class
```

Comment:
I can't see any reference in the doc to the labels used. Please document 1 = general snag and so on

Reply:
I'll get this in the documentation. Should be easy enough...